src/main/scala/lectures/functions/Fibonacci.scala

```diff
@@ -1,33 +1,35 @@
 package lectures.functions
 
+import scala.annotation.tailrec
+
 /**
-  * Цель упражнения: вычислить 9 - е число Фибоначчи
-  * Для этого раскомментируйте строчку в методе fibs и исправьте ошибку компиляции.
+  * Цель упражнения, вычислить 9 - е число Фибоначчи
+  * Для этого, раскомментируйте строчку в методе fibs и исправьте ошибку компиляции.
   *
   * Данная реализация вычисления чисел фибоначчи крайне не оптимальна (имеет показатеьную сложность O(a.pow(n)) )
-  * Для того, что бы в этом убедиться, Вы можете раскомментировать
+  * Для того, что бы в этом убедиться Вы можете раскомментировать
   * строчку с вычислением 1000-ого числа фибоначчи
   *
   */
 object Fibonacci extends App {
 
   // Task 2
-  def fibs(num: Int) = {
-    if (num == 1) 1
-    if (num == 2) 1
-    //fibs(num - 1) + fibs(num - 2)
+  def fibs(num: Int): Int = {
+    if (num == 1) 1 else
+    if (num == 2) 1 else
+    fibs(num - 1) + fibs(num - 2)
   }
 
   println(fibs(9))
   //println(fibs(1000))
 }
 
 /**
-  * Цель упражнения: используя приемы динамического программирования,
-  * реализовать более оптимальный алгоритм подсчета чисел фибоначчи
+  * Цель упражнения, используя приемы динамического программирования
+  * реаилзовать более оптимальный алгоритм подсчета чисел фибоначчи
   * Для этого нужно реализовать функцию fibsImpl.
   * Сигнатуру функции Вы можете расширять по своему усмотрению,
-  * но реализация должна удовлетворять следующим требованиям
+  * но реализация должна удовлетварять следующим требованиям
```

Comment:
https://ru.wiktionary.org/wiki/%D1%83%D0%B4%D0%BE%D0%B2%D0%BB%D0%B5%D1%82%D0%B2%D0%BE%D1%80%D1%8F%D1%82%D1%8C
:)